Gems/ScriptCanvas/Code/Editor/Assets/ScriptCanvasFileHandling.cpp

```diff
@@ -295,6 +303,13 @@ namespace ScriptCanvasEditor
                 , serializeContext
                 , AZ::ObjectStream::FilterDescriptor(nullptr, AZ::ObjectStream::FILTERFLAG_IGNORE_UNKNOWN_CLASSES)))
             {
+                if (IsUsingLegacySpawnTypes(asString))
+                {
+                    return AZ::Failure(AZStd::string::format(
+                        "Script Canvas source asset is using legacy Spawnable nodes and needs to be updated by running the update script: "
+                        R"("py o3de\Gems\ScriptCanvas\SourceModificationScripts\UpdateSpawnableNodes.py %s")",
```

Comment:
The command line for python should be `python` not `py`, no?

Reply:
yeah you're right